python/cudf_polars/tests/test_slice.py

```diff
@@ -11,13 +11,14 @@
 
 @pytest.mark.parametrize(
     "offset",
-    [0, 1, 2],
+    [0, 1, 2, -10, -20, -1, -2, 20],
 )
 @pytest.mark.parametrize(
     "len",
-    [0, 2, 12],
+    [0, 2, 12, 11],
 )
-def test_slice(offset, len):
+@pytest.mark.parametrize("slice_pushdown", [False, True])
+def test_slice(offset, len, slice_pushdown):
```

Comment:
Avoid shadowing the Python built-in `len`. I would rename this parameter to `length` to match the Polars API: https://docs.pola.rs/api/python/stable/reference/dataframe/api/polars.DataFrame.slice.html